exo-run/src/app-runner.ls

```diff
@@ -26,17 +27,15 @@ class AppRunner extends EventEmitter
     @process = DockerCompose.run-all-images {@env, cwd: @docker-config-location, @write}, (exit-code) ~>
       | exit-code => return @shutdown error-message: 'Failed to run images'
 
-    online-texts = @_compile-online-text!
-    asynchronizer = new Asynchronizer Object.keys(online-texts)
-
-    for role, online-text of online-texts
-      let role, online-text
-        @process.wait (new RegExp(role + ".*" + online-text)), ~>
-          @logger.log {role, text: "'#{role}' is running"}
-          asynchronizer.check role
-
-    asynchronizer.then ~>
-      @write 'all services online'
+    @_compile-online-text (@online-texts) ~>
```

Comment:
Two things:
1) Every callback should have the format `(err, result) ->` or `(err) ->`
2) If you are building `@online-texts` as a global object, then it does not need to be returned here. So we can just do:
```
@_compile-online-text (err) ~>
  | err => throw err
  do something with @online-text...
```